ocrd_network/ocrd_network/processing_worker.py

```diff
@@ -0,0 +1,332 @@
+"""
+Abstraction for the Processing Server unit in this arch:
+https://user-images.githubusercontent.com/7795705/203554094-62ce135a-b367-49ba-9960-ffe1b7d39b2c.jpg
+
+Calls to native OCR-D processor should happen through
+the Processing Worker wrapper to hide low level details.
+According to the current requirements, each ProcessingWorker
+is a single OCR-D Processor instance.
+"""
+
+import json
+import logging
+from os import environ, getpid
+import requests
+from typing import Any, List
+
+import pika.spec
+import pika.adapters.blocking_connection
+
+from ocrd import Resolver
+from ocrd_utils import getLogger
+from ocrd.processor.helpers import run_cli, run_processor
+
+from .database import (
+    sync_initiate_database,
+    sync_db_get_workspace,
+    sync_set_processing_job_state
+)
+from .models import StateEnum
+from .rabbitmq_utils import (
+    OcrdProcessingMessage,
+    OcrdResultMessage,
+    RMQConsumer,
+    RMQPublisher
+)
+from .utils import (
+    verify_database_uri,
+    verify_and_parse_mq_uri
+)
+
+try:
+    # This env variable must be set before importing from Keras
+    environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+    from tensorflow.keras.utils import disable_interactive_logging
+    # Enabled interactive logging throws an exception
+    # due to a call of sys.stdout.flush()
+    disable_interactive_logging()
```

Comment:
Can you add a `TODO` comment here to fix this when refactoring the logging please?